src/curated-corpus/pages/ProspectingPage/ProspectingPage.tsx

```diff
@@ -692,20 +705,50 @@ export const ProspectingPage: React.FC = (): JSX.Element => {
                 )}
               </Box>
             </Grid>
+            <Grid item xs={12}>
+              <ProspectPublisherFilter
+                filterByPublisher={filterByPublisher}
+                setFilterByPublisher={setFilterByPublisher}
+                onChange={toggleExcludePublisherSwitch}
+                excludePublisherSwitch={excludePublisherSwitch}
+              />
+            </Grid>
           </Grid>
 
           {!prospects && <HandleApiResponse loading={loading} error={error} />}
           {showEmptyState && <EmptyState />}
 
           {prospects &&
-            prospects.map((prospect) => {
+            prospects.map((prospect: Prospect) => {
               if (prospect.rejectedCorpusItem) {
                 // If an item was rejected, lets just not show it
                 return;
               }
 
-              // if the prospect has an approvedItem meaning it is in the corpus
+              // Filter the prospects already pre-loaded on the page
+              if (filterByPublisher.length > 2 && prospect.publisher) {
+                // Exclude prospects that match the filter
+                if (
+                  excludePublisherSwitch &&
+                  prospect.publisher
+                    .toLowerCase()
+                    .includes(filterByPublisher.toLowerCase())
+                ) {
+                  return;
+                }
+                // Include prospects that match the filter
+                else if (
+                  !excludePublisherSwitch &&
+                  !prospect.publisher
+                    .toLowerCase()
+                    .includes(filterByPublisher.toLowerCase())
+                ) {
+                  return;
+                }
+              }
+
```

Comment:
We could extract the above logic into a separate helper function but I guess we'll have to do that as part of the larger refactor for this component that we are waiting for 😄